include/pybind11/numpy.h

```diff
@@ -120,6 +120,25 @@ inline numpy_internals &get_numpy_internals() {
     return *ptr;
 }
 
+PYBIND11_NOINLINE module_ import_numpy_core_submodule(const char *submodule_name) {
+    try {
+        return module_::import((std::string("numpy._core.") + submodule_name).c_str());
```

Comment:
Could you please add a (terse) comment here explaining when this is expected to work, and below similarly?

Rough idea:

```
// numpy._core was established with numpy PR ...
```

```
// numpy.core was the import needed up until numpy release ...
```

Reply:
Sure! I added a comment.